packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -1,17 +1,23 @@
 <template>
-	<div ref="sqlEditor" class="ph-no-capture"></div>
+	<div :class="[$style.sqlEditor, 'sql-editor']" v-click-outside="onBlur">
```

Comment:
We needed one of them not to be not-module class so it can be referenced from our global css. But now that we removed SQL autocomplete notice, this is not needed any more.